cli/test/03-whoami.js

```diff
@@ -0,0 +1,80 @@
+import test from 'ava';
+import td from 'testdouble';
+import nock from 'nock';
+
+import whoami from '../lib/commands/whoami';
+
+// Have to run backend-involving tests serially so we are not nocking the
+// same URLs (maybe there is a better way?)
```

Comment:
You could set things up so that each test receives a different `registry` config value, maybe?

Reply:
Omg, yes! I have an idea how to do it now, thanks!

Reply:
Maybe setting it up like this would help? https://github.com/entropic-dev/entropic/pull/186/files#diff-b756986a1c0788ff75bc370380260e31R12